pkg/proxy/envvar_test.go

```diff
@@ -0,0 +1,90 @@
+package proxy_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/operator-framework/operator-marketplace/pkg/proxy"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	NO_Proxy    = "NO_PROXY"
+	HTTP_Proxy  = "HTTP_PROXY"
+	HTTPS_Proxy = "HTTPS_PROXY"
+)
+
+func TestGetOperatorEnvVars(t *testing.T) {
```

Comment:
Nit: I am not sure what feature this test is exercising that is not exercised in `TestSetEnvVars()`

Reply:
That's a good point, should I remove the test?

Reply:
I'll remove the test.